x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/sender/RequestExecutorService.java

```diff
@@ -213,15 +192,10 @@ public void start() {
             startCleanupTask();
             signalStartInitiated();
 
-            while (isShutdown() == false) {
-                handleTasks();
-            }
-        } catch (InterruptedException e) {
```

Comment:
Since we're not sleeping and we're not using a long lived thread we don't need to catch the interrupted exception.